pandas/core/frame.py

```diff
@@ -4969,6 +4969,39 @@ def f(x):
 
         return Series(result, index=labels)
 
+    def nunique(self, axis=0, dropna=True):
+        """
+        Return Series with number of distinct observations over requested
+        axis.
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            0 or 'index' for row-wise, 1 or 'columns' for column-wise
```

Comment:
@xflr6 can you make the doc-string references to the axis consistent with other methods

e.g. example from another method.
```
        axis : {0 or 'index', 1 or 'columns'}, default 0
            Sort index/rows versus columns
```

I think you can simply drop the 2nd line of the axis parm